bin/setenv-docker-customize.sh

```diff
@@ -154,6 +155,11 @@ EXO_ES_URL="${EXO_ES_SCHEME}://${EXO_ES_HOST}:${EXO_ES_PORT}"
 
 [ -z "${EXO_REGISTRATION}" ] && EXO_REGISTRATION="true"
 
+[ -z "${EXO_CLUSTER}" ] && EXO_CLUSTER="false"
+[ -z "${EXO_CLUSTER_NAME}" ] && EXO_CLUSTER_NAME="${HOSTNAME}"
```

Comment:
In fact, it's the cluster *node* name. I have changed this variable name to EXO_CLUSTER_NODE_NAME